packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -253,13 +254,9 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
    */
   handleInputValueChange = (event) => {
     event.stopPropagation();
-    const updatedValue = event.detail.newInputValue;
-    const isSelectionReset = event.detail.resetSelection;
-    if (this.standaloneSearchBox?.state?.value !== updatedValue) {
-      if (isSelectionReset) {
-        this.quanticSearchBoxInput.resetSelection();
-      }
-      this.standaloneSearchBox.updateText(updatedValue);
+    const newValue = event.detail.value;
+    if (this.standaloneSearchBox?.state?.value !== newValue) {
```

Comment:
we do need reseting the selections and we are still doing that, just the logic now is directly in the `QuanticSearchBoxInput`  component internally  instead of what we had before having to call `this.quanticSearchBoxInput.resetSelection()`